framework/CHANGELOG.md

```diff
@@ -49,7 +49,7 @@ Yii Framework 2 Change Log
 - Enh #12988: Changed `textarea` method within the `yii\helpers\BaseHtml` class to allow users to control whether html entities found within `$value` will be double-encoded or not (cyphix333)
 - Enh #13074: Improved `\yii\log\SyslogTarget` with `$options` to be able to change the default `openlog` options. (timbeks)
 - Enh: Added constants for specifying `yii\validators\CompareValidator::$type` (cebe)
-
+- Enh: #10896 Select only primary key when counting records in UniqueValidator
```

Comment:
your name is missing at the end of the line :) Also it would be good to order items in changelog by issue number. This avoids merge problems.